server/embed/config.go

```diff
@@ -178,6 +179,10 @@ type Config struct {
 	MaxTxnOps           uint   `json:"max-txn-ops"`
 	MaxRequestBytes     uint   `json:"max-request-bytes"`
 
+	// This is the time duration after which a warning is generated if applying request
```

Comment:
By convention, the comment should start with the flag name, i.e., s/This/ExperimentalWarningApplyDuration. Otherwise the goword checking will complain: https://travis-ci.com/github/etcd-io/etcd/jobs/440054426#L442

Reply:
@jingyih Done